src/main/scala/search/gmosnorth/GmosNorthFilterSelector.scala

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package basic.search.gmosnorth
+
+import cats.implicits._
+import gem.enum._
+import gem.enum.GmosNorthFpu._
+import gem.enum.GmosNorthDisperser._
+import gem.enum.GmosNorthFilter._
+
+object GmosNorthFilterSelector {
+
+  /**
+   * Given a disperser, fpu, and wavelength in nanometers, find the set of appopriate blocking
+   * filter(s) for spectroscopy. An empty set means there is no solution. A set containing `None`
+   * means that a filter is not necessary. Filters do not overlap in practice but if a wavelength
+   * falls on the exact boundary both filters will be selected.
+   */
+  def selectBlocking(disperser: GmosNorthDisperser, fpu: GmosNorthFpu, nm: Int): Set[Option[GmosNorthFilter]] =
```

Comment:
Why not `nm: Wavelength`?

Reply:
But I guess I could change the API here instead of converting externally. I'll do that.

Reply:
If we want to update the library, in `ocs3` given a `Wavelength` you can convert to `nm` via `Wavelength.fromNanometers.reverseGet(wavelength)`.  `Wavelength` is also ordered so we could express the selection table directly in terms of `Wavelength` as well and not convert anything?

Reply:
Yeah I'm doing that now.